migrations/versions/1020_add_buyer_email_domain_table.py

```diff
@@ -0,0 +1,33 @@
+"""Add buyer email domain table
+
+Revision ID: 1020
+Revises: 1010
+Create Date: 2017-10-10 15:18:22.683693
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1020'
+down_revision = '1010'
+
+
+def upgrade():
+    op.create_table(
+        'buyer_email_domains',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('domain_name', sa.String(), nullable=False),
+        sa.PrimaryKeyConstraint('id', name=op.f('buyer_email_domains_pkey'))
+    )
+    op.create_unique_constraint(
+        op.f('uq_buyer_email_domains_domain_name'), 'buyer_email_domains', ['domain_name']
```

Comment:
Not sure about the naming of the constraint here, or about Postgres implementation of unique constraints, but running the migration we get  a listing under "Indexes" - all our other indexes are prefixed with `ix_` rather than `uq_`.  I have no idea if this has any broader implications than consistency/tidiness... @allait might know more/have opinions on this.  There was something sometime about a pattern that gets used to generate these names, but I can't really remember the details.

```
Indexes:
    "buyer_email_domains_pkey" PRIMARY KEY, btree (id)
    "uq_buyer_email_domains_domain_name" UNIQUE CONSTRAINT, btree (domain_name)
```

Reply:
I'm not saying there's anything wrong, just I'd like another opinion before approving.

Reply:
The name was autogenerated by Alembic, if that makes any difference?

Reply:
Yeah, I think that means it's probably the right name :)